larq_compute_engine/core/bgemm_kernels_arm64.h

```diff
@@ -310,6 +320,16 @@ void BinaryKernelNeonOutOfOrder32BP4x4(
       "ld1 {v6.4s}, [%[rhs_ptr]], #16\n"
       "ld1 {v7.4s}, [%[rhs_ptr]], #16\n"
 
+      // Perform the backtransformation (in int32)
```

Comment:
how do we cover the cases that backtransformation needs to be done in float?

Reply:
See above post. With the `int8` case in mind, I'd rather always do the backtransform as `int32` so its always exact.